examples/ses/template.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: ses.aws.crossplane.io/v1alpha1
+kind: Template
+metadata:
+    name: example
+spec:
+    forProvider:
+        region: us-east-2
+        template:
+            templateName: example
```

Comment:
I tested updating `spec.forProvider.template.templateName` on an existing Template resource and received the following error:

```
2021-07-23T15:24:19.702-0400    DEBUG   provider-aws    Cannot update external resource {"controller": "managed/template.ses.aws.crossplane.io", "request": "/example", "uid": "b15f8ce9-c765-4543-bf1c-435caaaa8bc6", "version": "50082", "external-name": "example"}
2021-07-23T15:24:19.702-0400    DEBUG   controller-runtime.manager.events       Warning {"object": {"kind":"Template","name":"example","uid":"b15f8ce9-c765-4543-bf1c-435caaaa8bc6","apiVersion":"ses.aws.crossplane.io/v1alpha1","resourceVersion":"50082"}, "reason": "CannotUpdateExternalResource", "message": "cannot update Template in AWS: TemplateDoesNotExist: Template example2 does not exist."}
```